web/pages/admin/book/currentbabybook.tsx

```diff
@@ -0,0 +1,74 @@
+import UploadButton from "@components/BabyBook/UploadButton";
+
+
+export default function CurrentBabyBookScreen() {
+    
+    return (
+        <div className="flex flex-col w-full h-full items-center">
+            <div className="flex flex-col bg-custom-background w-full h-20 justify-center items-center">
+                <div className="flex flex-row w-full items-center">
```

Comment:
Make the BabyTopBar into a component so it can be reused across the pages instead of having to be copies and pasted every time, especially if anything needs to change with it!